examples/GetStarted/README.md

```diff
@@ -0,0 +1,12 @@
+## Vision SDK Get Started example
+This project contains basic logic to set up and rum [Vision SDK](https://www.mapbox.com/android-docs/vision/overview/). Also it shows different working modes of [VisionView](https://github.com/mapbox/mapbox-vision-android/blob/dev/MapboxVision/src/main/java/com/mapbox/vision/view/VisionView.kt) class.
+
+![MacDown Screenshot](images/get_started.gif)
+
+### Running locally
```

Comment:
Do you think its worth to have common prerequisite/setup instruction page for all the examples? Don't like the idea to copy/paste this info every time.